### DISCUSSION TITLE
Windows Package Manager v1.0.11692

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1215] [Releases]
This release fixes a few bugs. The first one is related to users and installers executing [Ctrl]+[c]. The next is a fallback when Delivery Optimization encounters failures. The last one addresses permissions inherited from the directory installers are saved to, and may also help with some installers due to long path names.

## Bugs

#189 Discord does not install
#1060 Fails to update program, error 0x801901a0
#1076 winget install results in 0x80d03002 : unknown error

## Notes

>Over the coming weeks we will be making changes to version numbers for the App Installer shipped in the Microsoft Store. We will be making some space for servicing agility so we can continue to iterate quickly with our preview releases. This version will be published to all Windows Insider channels and the Windows Package Manager Insiders.

>Going forward, development releases will be published to Windows Insider Dev channel and Windows Package Manager Insiders. We will leverage Windows Insider Beta channel for servicing issues, and the Windows Insider Release Preview channel for future release candidates.

>If you want the latest preview releases, there will be three choices.
>1. Join the [Windows Package Manager Insider program](http://aka.ms/winget-InsiderProgram).
>2. Run a Windows Insider Dev channel build.
>3. Download the preview releases from GitHub.

>Any users on released versions of Windows who have downloaded releases from this GitHub repository should ensure they download and install this release to ensure continuity of access to the Windows Package Manager. With this release we are discontinuing support for versions of Windows 10 prior to version 1809.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.0.11692'>Windows Package Manager v1.0.11692</a>.</em>

**Comment by adrianghc** [comment]
I'm not sure I understand how you are pushing these now. The description makes it sound like Release Preview will get release candidates, Dev and Package Manager Insiders will get the actual releases (so later than RP?) and beta will just get servicing updates? That doesn't seem right.

**Reply by denelon** [reply]
@adrianghc thanks for the comment. I updated the notes.

**Reply by adrianghc** [reply]
Thank you for the quick reaction. One thing is still not quite clear, will beta receive faster updates than RP or the other way round?

**Reply by denelon** [reply]
DEV - Most Updates (Fastest) - As often as we have bugs or features to test
BETA - Less Frequent Updates -  Maybe monthly, will lag behind DEV
Release Preview - Least Frequent Updates - Maybe quarterly, before scheduled stable releases

**Reply by adrianghc** [reply]
Will updates appear faster in dev than they have been appearing until now as GitHub releases? Will GitHub releases mirror the update cadence of dev or of beta?

**Reply by denelon** [reply]
DEV and GitHub will likely be pretty similar.
There are several bug fixes and minor improvements to make. We may see an increased cadence at GitHub and with DEV. 

BETA would likely be a month or so behind, and might not get every GitHub/DEV release.

**Reply by adrianghc** [reply]
Alright, thank you for the explanation!

**Reply by github-account1111** [reply]
>DEV - Most Updates (Fastest) - As often as we have bugs or features to test

Has this been scraped? Last update was three months ago.
If dev/github updates are so infrequent, then I can only imagine what beta and release are like.

**Reply by denelon** [reply]
https://github.com/microsoft/winget-cli/releases

9/16/2021 - Windows Package Manager v1.0.12576
6/24/2021 - Windows Package Manager v1.0.11692
5/26/2021 - Windows Package Manager v1.0.11451

No, but we did have an extended period between the last two releases. Many of the changes were service side and REST API side rather than client side. The team has also been working on [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create).

**Reply by github-account1111** [reply]
Oh I did not see the one from 9/16. Somehow it's not listed if looking from the main readme page.

Still that's at best once a month for the dev channel, when that is the schedule intended for beta. I was just confused and thinking maybe I was looking in the wrong place.

**Reply by denelon** [reply]
No, we just haven't been releasing for every change. We've had a lot of work to do on the back end to support differences for different flavors of the OS. I hope to get to a point where we are able to release more frequently. Thanks for sharing the concern. It helps to know if the community wants us to release smaller changes more frequently.

**Comment by adrianghc** [comment]
Just fyi, not sure if this is as intended but I still haven't received this new version through the Microsoft Store despite being a Windows Package Manager Insider.

**Reply by denelon** [reply]
@adrianghc you can install the latest release here from GitHub. Just double click the .msixbundle.

**Reply by adrianghc** [reply]
I know I can, I was just wondering why it was not showing up on the Store.

**Reply by denelon** [reply]
@adrianghc, It's likely the delay in adding your e-mail address to the group. If you want to send me an e-mail with the address you added, I can check. That's not the only reason, just the most likely.

**Reply by albertopasqualetto** [reply]
Also I didn't have the update, but I installed the previous one trough the store, so I think we should wait a bit more.

Edit: I just received it!
